src/rust/madara/template/src/template.rs

```diff
@@ -257,6 +260,23 @@ pub fn get_search_result(data: MadaraSiteData, url: String) -> Result<MangaPageR
 }
 
 pub fn get_series_page(data: MadaraSiteData, listing: &str, page: i32) -> Result<MangaPageResult> {
+	// Monkeypatch for now until the source api rewrite
+	if !data.use_ajax_listing {
+		let listing = match listing {
+			"_wp_manga_views" => "views",
+			"_wp_manga_week_views_value" => "trending",
+			"_latest_update" => "latest",
+			_ => "latest",
+		};
+
+		let url = format!(
+			"https://manhuaplus.com/page/{}?s&post_type=wp-manga&m_orderby={}",
```

Comment:
wait, shouldn't this be using the template base url, not just manhuaplus?

Reply:
oh i completely missed that